.env.example

```diff
@@ -55,5 +55,18 @@ MAIL_TRANSPORT_SMTP_SECURE=false
 MAIL_TRANSPORT_SMTP_AUTH_USER=myUser
 MAIL_TRANSPORT_SMTP_AUTH_PWD=password
 
+# DKIM (DomainKeys Identified Mail) signature related configuration
+# You need all three if you want to use DKIM, but you don't have to set them if you
+# do not want to add the DKIM signature to the emails
```

Comment:
It applies to all email strategies? If so, you have nothing to add here, but if it applies to only one, please add a comment

Reply:
Yeah, all